tests/unittests/test_minor_version.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+from unittest import mock
+
+from tap_quickbooks import QuickbooksClient
+from tap_quickbooks.streams import Accounts
+
+@mock.patch('tap_quickbooks.client.OAuth2Session.request')
+@mock.patch('tap_quickbooks.client.QuickbooksClient.get')
+class TestMinorVersion(unittest.TestCase):
+    '''
+    Tests to verify minor_version works as expected.
+    '''
+
+    def test_minor_version(self,mocked_get, mocked_request):
```

Comment:
Is this the correct order for the decorators? If so, is there a way to make this more clear? each test function takes arguments in the opposite order of the decorators

Reply:
Yes, the order is correct.
For @mock.patch decorators, it will follow the bottom-up approach. So the first function arg will relate to the lowermost patch decorator.

Any suggestions to make it more clear?